pkg/attestation/crafter/runners/githubaction.go

```diff
@@ -68,3 +82,19 @@ func (r *GitHubAction) RunURI() (url string) {
 func (r *GitHubAction) ResolveEnvVars() (map[string]string, []*error) {
 	return resolveEnvVars(r.ListEnvVars())
 }
+
+func (r *GitHubAction) RunnerEnvironment(ctx context.Context) string {
+	return r.oidcClient.RunnerEnvironment(ctx)
+}
+
+func (r *GitHubAction) WorkflowFilePath(ctx context.Context) string {
+	return r.oidcClient.WorkflowFilePath(ctx)
+}
+
+func (r *GitHubAction) IsHosted(ctx context.Context) bool {
+	return r.oidcClient.IsHosted(ctx)
```

Comment:
coming back to this, I think this should return `true` always, as the others. I cannot think of a case where this would be false for Github.

Reply:
I agree to some extent, I can make it always `true`, but then exporting the right env variables in a local environment will make it also `true` right? Maybe I'm focused on that too much?

Reply:
Changed :)